README.md

```diff
@@ -1890,7 +1890,7 @@ The Plot.windowX and Plot.windowY transforms compute a moving window around each
 * **k** - the window size (the number of elements in the window)
 * **anchor** - how to align the window: *start*, *middle*, or *end*
 * **reduce** - the aggregation method (window reducer)
-* **strict** - if true, disallow window truncation; defaults to false
+* **strict** - if true, disallow window truncation and skips invalid values; defaults to false
```

Comment:
I think we can say something more precise than “skips invalid values”, since we can’t really “skip” them (we have to do _something_). Something like: if strict, if any value in the input window is invalid, including when the window extends beyond the start or end of the series, the corresponding output value is undefined. I’m tempted to say the output value is NaN rather than undefined, but that’s not true in the case of the _first_ and _last_ reducers since these are non-numeric.